internal/captain/command.go

```diff
@@ -33,6 +33,7 @@ import (
 
 func init() {
 	configMediator.NewRule(constants.UnstableConfig, configMediator.Bool, configMediator.EmptyEvent, configMediator.EmptyEvent)
+	configMediator.NewRule(constants.AutoUpdateConfigKey, configMediator.Bool, configMediator.EmptyEvent, configMediator.EmptyEvent)
```

Comment:
This should go in `cmd/state/autoupdate.go`. The reason we architected it this way is so that the package that utilizes the key is also the one registering it.

Reply:
What is this magic? I can just add an `init()` function to *autoupdate.go* and it magically gets called? I mean, I just tried it and it appears to work, but ??!!